apps/i18n/common/en_us.json

```diff
@@ -1966,12 +1966,13 @@
   "plSectionsRegionalPartnerApplicationTitle": "Application Dashboard",
   "plSectionsRegionalPartnerApplicationDesc": "Manage applications for the current application season.",
   "plSectionsRegionalPartnerApplicationButton": "Manage applications",
-  "plSectionsRegionalPartnerWorkshopTitle": "Workshop Dashboard",
-  "plSectionsRegionalPartnerWorkshopDesc": "Manage upcoming, in progress, and past workshops.",
-  "plSectionsRegionalPartnerWorkshopButton": "View workshop dashboard",
```

Comment:
Since this component and the Regional Partner Resources section use the same strings, I updated the names to be more universal. Since these strings were just created I feel like changing the names won't affect the translation timeline too much but I'm open to pushback if I should just use the existing names even though they'll say "RegionalPartner" in them

Reply:
They might not have even made it to Crowdin yet and this page in general isn't high priority for translation. So I think this makes sense!